spec/spec_helper.rb

```diff
@@ -130,29 +123,7 @@ def main_app
 
   config.include Devise::TestHelpers, type: :controller
   config.include Devise::TestHelpers, type: :helper
-  config.include EngineRoutes, type: :controller
   config.include Warden::Test::Helpers, type: :feature
   config.after(:each, type: :feature) { Warden.test_reset! }
   config.infer_spec_type_from_file_location!
 end
-module FactoryGirl
-  def self.find_or_create(handle, by=:email)
```

Comment:
This is a weird addition to FactoryGirl. Would argue you shouldn't be using factory girl to lookup a object from the database? Especially if we are clearing out the database after every example. Feel like you should have a bit of understanding of what your trying to do instead of just calling `FactoryGirl.find_or_create()` everywhere in our test suite. Can we get away with a build? a build_stubbed? Then we should use create?